index.html

```diff
@@ -242,6 +244,52 @@ <h4>Skills</h4>
                 </div>
             </div>
         </section>
+        <!--Contact form section-->
+        <section class="contact-me">
+            <!--contact me section main text-->
+            <div class="contactme-text">
+                <h1 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            </div>
+            <!--form div-->
+            <div class="form-outline">
+                <form action="https://formspree.io/f/xeqwblvq" method="POST">
+                  <ul class="contactme-ul">
+                    <li class="name">
```

Comment:
- [ ] Kindly remove these `ul` and `li` tags from your form because the **ul** tag is meant for containing a list of items but here you have the form which has some input fields not the list of items. As I can understand for the sake of design it seems ok but semantically, it is not correct at all.